README.md

```diff
@@ -168,9 +191,7 @@ and [create a simulation job](https://docs.aws.amazon.com/robomaker/latest/dg/cr
 
 ## Generate map for your world
 
-Procedurally generate an occupancy map for any gazebo world. This map can then be plugged into your navigation stack to navigate a robot in your world. 
-
-Note: This is an OPTIONAL functionality and is NOT required for using any functionalities listed above.
+Procedurally generate an occupancy map for any gazebo world. This map can then be plugged into your navigation stack to navigate a robot in Worldforge worlds. For other aws-robotics worlds, this functionality is currently optional. 
```

Comment:
Is this actually optional for the other `aws-robotics` worlds? Or is it that they already have occupancy maps available?

Reply:
yep they already have occupancy maps available.. in the use-case of this README(navigation+metric tracking via cloudwatch), generating occupancy maps for these worlds is optional

Reply:
Let's explicitly state that it's optional because they already have occupancy maps (albeit lower quality), or users may be confused as to why navigation is possible in these particular worlds without needing to generate occupancy maps.

It's also not the navigation functionality that's optional for the `aws-robotics` worlds, but rather it's carrying out the procedure of generating occupancy maps for them that's optional, right?